### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by noamrubin22**
readme updated

### CODE REVIEW

**Review by LunaNiermann — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -63,69 +74,101 @@ This will set up a local development envirenment, getting you ready to work on w
...
+
+If you are not planning to work with the map or “search” you can skip this step, otherwise it is required.
+
+* Install Elasticsearch (https://www.elastic.co/downloads/elasticsearch)
+  * For macOS: 
+   * `brew tap elastic/tap`
+   * `brew install elastic/tap/elasticsearch-full`
+* Switch between haystack settings in the settings.py file by changing the comment section:
+
+`
```
Comment (saschasommer) [inline README.md]:
````suggestion
```
````

File: README.md [hunk 2]
```diff
@@ -63,69 +74,101 @@ This will set up a local development envirenment, getting you ready to work on w
...
+    # enable this haystack setting if you have actually set up elastic search on your system
+
+    HAYSTACK_CONNECTIONS = {
+        'default': {
+            'ENGINE': 'cosinnus.backends.RobustElasticSearchEngine',  # replaces 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine',
+            'URL': 'http://127.0.0.1:9200/',
+            'INDEX_NAME': 'wechange',
+        },
+    }
+`
```
Comment (saschasommer) [inline README.md]:
````suggestion
```
````

**Review by saschasommer — COMMENTED** [review]
Added missing backticks for python code example as suggestions.

**Comment by noamrubin22** [comment]
added your suggestions

**Review by saschasommer — APPROVED** [review]

**Pull request merged**